### PR TITLE
*: update `remove-meta`

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- remove `remove-meta` item in task config
- add `remove-meta` flag for `start-task`

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

File: zh/task-configuration-file.md [hunk 1]
```diff
@@ -81,13 +81,11 @@ mysql-instances:
 
 ## 修改任务配置
 
-在某些情况下需要更新任务配置内容，比如重置数据同步任务时设置了 `remove-meta` 为 `true` 和 `task-mode` 为 `all`，当数据同步任务重置完成后，需要更新任务配置文件中的 `remove-meta` 设置为 `false`，防止下一次启动任务的时候重新同步任务。
-
 因为 DM 集群会持久化保存任务配置，所以修改任务配置需要通过 `stop-task`、`start-task` 将修改后的配置更新到 DM 集群中，如果直接修改任务配置文件，但是不重启任务，配置变更不会生效，DM 集群重启时仍然会读取之前保存的任务配置。
 
-这里以修改 `remove-meta` 配置为例，举例说明任务配置修改步骤：
```
Comment (csuzhangxc) [inline zh/task-configuration-file.md:88]: `remove-meta` 这个已经从 task config 里移除了，所以这里换一个示例

**Comment by csuzhangxc** [comment]
@GMHDBJD @ireneontheway PTAL

**Review by GMHDBJD — COMMENTED** [review]
LGTM

**Comment by ireneontheway** [comment]
@TomShawn LGTM

File: en/create-task.md [hunk 2]
```diff
@@ -35,6 +36,7 @@ start-task [ -s "mysql-replica-01"] ./task.yaml
 
 - `-s`: (Optional) Specifies the MySQL source to execute `task.yaml`. If it is set, the command only starts the subtasks of the specified task on the MySQL source.
 - `config-file`: (Required) Specifies the file path of `task.yaml`.
+- `remove-meta`: (Optional) Specifies whether to remove the task's previous meta data when starting the task.
```
Comment (TomShawn) [inline en/create-task.md:39]:
```suggestion
- `remove-meta`: (Optional) Specifies whether to remove the task's previous metadata when starting the task.
```

File: zh/create-task.md [hunk 3]
```diff
@@ -44,6 +42,9 @@ start-task [ -s "mysql-replica-01"] ./task.yaml
 + `config-file`：
     - 必选
     - 指定 `task.yaml` 的文件路径
++ `remove-meta`:
+    - 可选
+    - 如果设置，则在启动指定任务之前会移除该任务之前存在的 metadata
```
Comment (TomShawn) [inline zh/create-task.md:47]: `启动指定任务之前` 还是 `启动指定任务时`？

Comment (csuzhangxc) [reply zh/create-task.md]: 在 https://github.com/pingcap/docs-dm/pull/294/commits/0b45a50d453dab39953c57c37390aa831bddec0d 改了

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**